src/Verismith.hs

```diff
@@ -283,6 +283,25 @@ handleOpts (DistanceOpt v1 v2) = do
   src2 <- parseSourceInfoFile (T.pack v2) (toTextIgnore v2)
   let d = distance src1 src2
   putStrLn ("Distance: " <> show d)
+handleOpts (Equiv o v1 v2 top checker) = do
+  datadir <- getDataDir
+  src <- parseSourceInfoFile top (toTextIgnore v1) :: IO (SourceInfo ())
+  _ <- shelly $ do
+    mkdir o
+    cp v1 (o </> fn1)
+    cp v2 (o </> fn2)
+  res <- shelly . runResultT $ pop o $ do
+      runEquiv checker (toFP datadir) (mkid fn1) (mkid fn2) src
+  case res of
+    Pass _ -> putStrLn "Equivalence check passed"
+    Fail (EquivFail _) -> putStrLn "Equivalence check failed"
+    Fail TimeoutError -> putStrLn "Equivalence check timed out"
+    Fail _ -> putStrLn "Equivalence check error"
+  return ()
```

Comment:
I guess you don't need this line, `putStrLn` returns type `IO ()` already.

Reply:
Removed, same with the useless binding `_ <- shelly [...]` at line 289.